2996_reflection/reflection.md

```diff
@@ -3810,11 +3919,18 @@ constexpr bool b3 = tfn<3>();
   // error: the invocation of make_decl(R) in the requires clause produces an
   // injected declaration but is not plainly constant-evaluated
 
-template <typename> struct S {
+template <typename> struct T {
   static constexpr bool b4 = make_decl(4);
     // error: target scope of the injected declaration is not
     // instantatiation-sequenced with the expression
 }
+
+struct S;
+consteval bool complete_decl();
+  // produces an injected declaration completing S
+
+constexpr auto v = (complete_decl(), 1) + sizeof(^^S);
+  // ill-formed, no diagnostic required
```

Comment:
Don't need the `(x, 1)`, can just `complete_decl() + sizeof(^^S)`.

Reply:
Done.